ecommerce-ui/src/components/ShoppingCart.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import ShoppingCartItem from './ShoppingCartItem';
+
+// feeds RentalPage
+
+class ShoppingCart extends Component {
+
+  render() {
+    return (
+        <form>
+          <div className="cart-all">
+          <h1>Properties Rented:</h1>
+           {this.props.selectedRentals.map((airbnb, index) =>
+            <ShoppingCartItem
```

Comment:
Lines 15-19 are a function that should return something, i.e.
{return <ShoppingCartItem.../>}